contract-deployment/contractDeploymentTools.go

```diff
@@ -153,7 +25,10 @@ func main() {
 		if len(os.Args) != 3 {
 			log.Panic("Invalid argument count. Must provide JSON file containing contract bytecode.")
 		}
-		constructKeylessTransaction(os.Args[2])
+		_, _, _, err := tests.ConstructKeylessTransaction(os.Args[2], true)
```

Comment:
It feels a little odd to be using a utility from the `tests` package in the `main` package of another application. Could we move it to a `delpoyment-utils` package under `contract-deployment` still maybe?

Reply:
Moved to `package deployment_utils` under `contract-deployment`